code/modules/power/multiz.dm

```diff
@@ -1,144 +1,21 @@
-#define RELAY_OK 1
-#define RELAY_ADD_CABLE 2
-#define RELAY_ADD_METAL 3
-
-/obj/machinery/power/deck_relay //This bridges powernets
-	name = "Multi-deck power adapter"
-	desc = "A huge bundle of double insulated cabling which seems to run up into the ceiling."
+/obj/structure/cable/multilayer/MultiZ //This bridges powernets
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#type-paths-must-be-lowercase
No uppercase in typepaths

Reply:
Ok